src/configuration-component.ts

```diff
@@ -133,6 +133,7 @@ export class ConfigurationComponent {
         <option value="icy-dark">Icy Dark</option>
         <option value="dark-blue">Dark Blue</option>
         <option value="photon-dark">Photon Dark</option>
+        <option value="hugo-terminal-orange">Hugo Terminal Orange</option>
```

Comment:
Could you move this directly beneath the github-dark-orange theme? Easier toggle between the two with the keyboard for comparison purposes.